includes/definitions/discovery/gaia.yaml

```diff
@@ -78,6 +79,26 @@ modules:
                         - { value: 0, descr: Up , graph: 2, generic: 0 }
                         - { value: 1, descr: Attention, graph: 1, generic: 3 }
                         - { value: 2, descr: Down, graph: 0, generic: 1 }
+                -
+                    oid: mglsGWState
+                    num_oid: '.1.3.6.1.4.1.2620.1.7.14.4.1.3.{{ $index }}'
+                    index: 'mglsGWState.{{ $index }}'
+                    descr: '{{ $mglsGWIP }}'
+                    group: 'Management Connected Gateways'
+                    value: mglsGWState
+                    states:
+                        - { value: 1, descr: Connected, graph: 1, generic: 0 }
+                        - { value: 0, descr: Disconnected, graph: 0, generic: 2 }
+                -
+                    oid: fwSXLStatus
+                    num_oid: '.1.3.6.1.4.1.2620.1.36.1.1.{{ $index }}'
+                    index: 'fwSXLStatus.{{ $index }}'
+                    descr: 'SecureXL current status'
+                    group: HA
+                    value: fwSXLStatus
+                    states:
+                        - { value: 1, descr: Enabled , graph: 0, generic: 0 }
+                        - { value: 0, descr: Disabled, graph: 1, generic: 3 }
```

Comment:
Same thing as for `haState`. 
`generic` could be changed to `2` instead of  `3` to make it an `Critical` instead of `Unknown`.

Reply:
As the SecureXL is an acceleration solution that maximizes performance, but if disabled will not be a critical problem, I think `1` `Warning` fits well